BGL/include/CGAL/boost/graph/named_params_helper.h

```diff
@@ -238,58 +234,53 @@ namespace CGAL {
   NP,                                                         \
   typename boost::property_map<TM, Final_tag >::const_type    \
   > ::type  type;                                             \
-  };
-
+};
+  
```

Comment:
Please, just use the following setting ![please](https://user-images.githubusercontent.com/5074170/75039785-aad52600-54b9-11ea-9516-04f29f8b118b.png)